src/drive/mobile/lib/reporter.js

```diff
@@ -16,41 +14,36 @@ const getAnalyticsUrl = () => {
 export const ANALYTICS_URL = getAnalyticsUrl()
 
 export const getReporterConfiguration = () => ({
-  shouldSendCallback: () => isEnabled,
+  shouldSendCallback: true,
   environment: __DEVELOPMENT__ ? 'development' : 'production'
 })
 
-export const configureReporter = enable => {
-  isEnabled = enable
+export const configureReporter = () => {
   Raven.config(ANALYTICS_URL, getReporterConfiguration()).install()
 }
 
-export const logException = err => {
+export const logException = (err, extraContext = null) => {
   return new Promise(resolve => {
-    Raven.captureException(err)
+    extraContext
+      ? Raven.captureException(err, { extra: extraContext })
+      : Raven.captureException(err)
```

Comment:
What happens if you just call `Raven.captureException(err, { extra: null })`?

Reply:
I don't know, that's why I preferred to handle the two cases differently ;)

Reply:
Ok! I don't mind either way, but I think it would be easier to set the default value to `{}` and remove the ternary.